portfolio/src/main/webapp/index.html

```diff
@@ -36,74 +46,89 @@ <h1 class="headerFont">Matthew Wong</h1>
+                    class="image" style="width:100%"></a>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="cardTitle">
+                <p class="fonts">Quizzler 📚</p>
+            </div>
+            <div>
+                <a href="https://github.com/MattWong-ca/MultipleChoiceQuiz"><img src="/images/QuizzlerSS.png" href="https://github.com/MattWong-ca/MultipleChoiceQuiz" width="242" height="451" class="image" style="width:100%"></a>
+            </div>
+        </div>
+
+        </div>
+
     </div>
     </br>
 
-    <!--Just For Fun-->
-    <div class="test">
-    <p class="headerFont">Just for Fun 😄</p>
-        
-        <!--Fun Facts-->
-        <p class="mediumFont"">Fun Fact:</p>
-        <p class="fonts">Click <a href="/hello">here</a> to see content created by server code. Click here to get a random message:</p>
-        <button class="fonts" onclick="addRandomGreeting()">Hello</button>
-        <div class="fonts" id="greeting-container"></div>
-
-        <!--Server Date-->
-        <p class="mediumFont"">Today is...</p>
-        <p class="fonts">Click this button to see the server's current date:</p>
-        <button onclick="showServerTime();" class=" fonts">Today is...</button>
-        <p class="fonts" id="date-container"></p>
-        <p class="fonts">Click <a href="/date">here</a> to see the server response directly.</p>
-        
-        <!--JSON--><!--client-side JavaScript and server-side Java-->
-        <p class="mediumFont"">JSON</p>
-        <p class="fonts">Click <a href="/experience">here</a> to see JSON. Click the button to see the server-side JSON:</p>
-        <button class="fonts" onclick="test();">JSON</button>
-        <p class="fonts" id="exp"></p>
+    <!--Learnings-->
+    <div class="wideDivContent">
+        <p class="headerFont">Learnings 📖</p>
+        <p class="mediumFont">Check out what I've been up to:</p>
+        <p class="fonts">• <a href="learnings.html">Summer '22</a></p>
     </div>
     </br>
 
     <!--Form-->
-    <div id="content">
-    <h1 class="headerFont">Contact me!</h1>
-    <form action="/form-handler" method="POST">
-        <p class="fonts">Send me a message here:</p>
-        <textarea name="text-input" class="fonts">Hi Matt,</textarea>
-        <br/>
-        <input type="submit" value="Send!" class="fonts"/>
-    </form>
+    <div class="centerDivContent">
+        <h1 class="headerFont">Contact me!</h1>
+        <form action="/form-handler" method="POST">
+            <p class="fonts">Send me a message here:</p>
+            <textarea name="text-input" class="fonts" placeholder="Your message"></textarea>
+            <br/>
+            <input type="submit" value="Send!" class="fonts" />
+        </form>
     </div>
 
-  </body>
-</html>
-
-<!--
-FAILED TAG LABEL CODE
+    <button class="fonts" onclick="fetchResponseList();">Click here to see form responses</button>
```

Comment:
optional: I recommend:
  1. Automatically fetching and displaying all messages on page load (or on form submission)
  2. Sort the responses by submission time, and put each one on it's own line. 

Currently its hard to see the responses button, and hard to read/parse the results.